app/concepts/api/v1/users/registrations/operation/create.rb

```diff
@@ -4,6 +4,7 @@ module Api::V1::Users::Registrations::Operation
   class Create < ApplicationOperation
     step :validate_contract
     step :create_user
+    step :send_email
```

Comment:
Rule of thumb: one step - one action. For your case it's: `create_email_token` and `send_email`.